### ISSUE TITLE
Add a guide to run the server in watch mode using `entr`

### ISSUE THREAD

**Issue by tusharmath** [tailcallhq/tailcallhq.github.io#33]
Write an engaging guide on how to run the server in watch mode using [entr]

[entr]: https://eradman.com/entrproject/

```bash
ls ./examples/jsonplaceholder.graphql | entr -r tc start ./examples/jsonplaceholder.graphql
```

Cover the following points —
- Need of running in watch mode
- Using entr
- Linking to installing entr
- Suggest some best practices

**Comment by Min2who** [comment]
can you assigned me this issue @tusharmath

**Comment by Min2who** [comment]
Hi @tusharmath we have to add the guides in the docs/guides/ section right and is there any specific name that we have to use

**Comment by tusharmath** [comment]
Yes in the guides section. Feel free to use any name, I think since this guide is for developers who are running tailcall in development mode, how about calling it `developer.md`?

**Comment by Min2who** [comment]
developer.md seems perfect

**Comment by Min2who** [comment]
I have created a PR please review it

**Comment by tusharmath** [comment]
fixed in #34
